protobuf/proto/crux.proto

```diff
@@ -824,13 +824,23 @@ enum NotificationType {
   TEAMS = 3;
 }
 
+enum NotificationEventType {
+  UNKNOWN_NOTIFICATION_EVENT_TYPE = 0;
```

Comment:
Does it add extra value to add the enum name for UNKNOWN? Prefer UNKNOWN instead of UNKNOWN_NOTIFICATION_EVENT_TYPE.

Reply:
You can't have Enum1.UNKNOWN and Enum2.UNKNOWN in the same proto file. Thats why we put postfixes on those values.

Reply:
Only if you are using the two enums in the same message. So if we don't use enums in the same message we can use simple UNKNOWN, the only question is which way is the better. 

1. Using simple UNKNOW where we can (results inconsistent enum definitions)
2. Using all UNKNOW with the enum type prefix like here (long types - cant workaround if enums in the same msg) 

What do you think?

Reply:
Talked in person with @balanceee 

Based on this official recommendation it's okay as we use it right now. https://cloud.google.com/apis/design/design_patterns#resource_view